sample_project/Assets/SampleViewer/Scripts/DisableTouchJoysticks.cs

```diff
@@ -0,0 +1,19 @@
+using UnityEngine;
+
+public class DisableTouchJoysticks : MonoBehaviour
+{
+    public static DisableTouchJoysticks instance;
+
+    private void Awake()
+    {
+        instance = this;
+        gameObject.SetActive(false);
+    }
+
+    public void ToggleCanvas(bool active)
+    {
+#if !UNITY_EDITOR && (UNITY_IOS || UNITY_ANDROID)
```

Comment:
Do you need to check for vision os here?

Reply:
No, because they don't get enabled in visions OS. We need completely different controls for the Camera in Vision OS. more traditional Touch Controls